### ISSUE TITLE
NativeScript 3 & Angular 4

### ISSUE THREAD

**Issue by ejsuncy** [AngularJSUtah/meetups-slc-group#6]
Come learn about NativeScript (the Angular analog of React Native). Build native iOS and Android apps with the TypeScript and [(banana in a box)] you already know and love! Other JS/TS app solutions like cordova & ionic focus on exposing native functionality to a webview, but NativeScript uses fully-native UI components, views, and transitions. There are many NativeScript plugins that expose device functionality in a cross-platform way, and if you need native functionality that doesn't yet have a plugin, you can still access the entire native API of your iOS/Android platform from your TypeScript/Angular app via FFI & marshalling.

**Comment by ejsuncy** [comment]
@kensnyder I'll have this prepared by May 2nd for my work presentation, and I'd be happy to present any time after that :)

**Comment by waltercodes** [comment]
Hey @ejsuncy are you by chance prepared to present tonight (May 23)? If not, no worries.

**Comment by ejsuncy** [comment]
@waltercodes no, most of my presentation is outdated since NS 3 was released. I could prepare it for the next one though. Just let me know!

**Comment by waltercodes** [comment]
@ejsuncy We still have an opening for June 27. Are you interested in speaking then?

**Comment by ejsuncy** [comment]
@waltercodes Yep! I'll plan on it.
